### ISSUE TITLE
When comment section is set to "show" and you backed to list choose another video, your newer video page will randomly scroll

### ISSUE THREAD

**Issue by fireattack** [ParticleCore/Particle#194]
It's hard to describe so I made a video.  https://youtu.be/SYAeEFs3mHs

Browser: Chrome Version 49.0.2612.0 dev-m (64-bit)
Extension version.

Repo:

1. Make sure you choose "show" for comment section option. I also set auto play as true, if it matters.
2. Open one of the video list, e.g. https://www.youtube.com/channel/UC2bkHVIDjXS7sgrgjFtzOXQ/videos
3. Open any video[1], double-check to make sure the comment section loaded.
4. Back to the list by press backspace or click the back button on the browser.
5. Click on another video and wait.

[1] I later realize it may not be able to repo with all the videos, so make sure you follow the videos I clicked in the video.

What happens: the page randomly scrolls down.

--- 

Edit: I can repo this on userscript verion with TM on Chrome. I *cannot* repo this on Firefox (xpi version). So I believe it's a Chrome only bug.

**Comment by ParticleCore** [comment]
Follow the troubleshooting guide to see whether or not this is caused by YT+ https://github.com/ParticleCore/Particle/wiki/Report-a-problem#troubleshooting

And if I remember correctly, Chrome does have a page position history method when going back and forth through the navigation history, so this is likely to be it.

**Comment by ParticleCore** [comment]
I am unable to replicate this with the same setup, but I do remember on occasion the page scrolling down when navigating back in history as long as I was on that lower section of the page (or near it).

**Comment by fireattack** [comment]
It's a shame I failed at basic troubleshooting.. I discovered this problem only *after* I enable comment section option and since I can repo it with new profile, I thought it's caused by YT+.. But nope, it's actually SPF's bug (before I start using YT+ I used other extension to disable SPF, that's why I didn't encounter this bug before).

I reported it to spf.js team (https://github.com/youtube/spfjs/issues/385) instead here. Thanks again and sorry for the inconvenience.
